pre_gauss:                  # Gaussian blur before edge detection
    apply: true
    ksize_x: 9
    ksize_y: 9
    sigma_x: 2
    sigma_y: 2

canny:                      # Edge detection parameters
    apply: true
    min_threshold: 80
    max_threshold: 120

post_gauss:                 # Gaussian blur before edge detection
    apply: true
    ksize_x: 9
    ksize_y: 9
    sigma_x: 2
    sigma_y: 2

hough:                      # Detect circles using hough transform
    dp: 2
    min_dist: 10
    param1: 100
    param2: 80
    min_radius: 20
    max_radius: 38

roi:                        # Region of interest to find circles
    x: 0
    y: 850
    width: 1900
    height: 400

visualize: false
